server/backend/db_backend.go

```diff
@@ -35,7 +35,8 @@ type MongoBackend struct {
 // New create new rpc client with given url
 func NewMongoClient(client *goclient.Client, host, dbName string, lgr *zap.Logger) (*MongoBackend, error) {
 	session, err := mgo.DialWithInfo(&mgo.DialInfo{
-		Addrs: []string{host},
+		Addrs:   []string{host},
+		Timeout: 10 * time.Second,
```

Comment:
Is this when the client quits on a long query?

Reply:
both (unfortunately without timeout it could stuck during start, but in the same time it also throwing timeout during long running queries - which is I won't say bad)

Reply:
set to 2 minutes